app/controllers/bigbluebutton_api_controller.rb

```diff
@@ -174,8 +174,15 @@ def create
 
     # Pass along all params except the built in rails ones and excluded_params
     uri = encode_bbb_uri('create', server.url, server.secret, pass_through_params(excluded_params))
+    callback_data = nil
 
     begin
+      if params['meta_bn-recording-ready-url'].present?
+        callback_attributes = { recording_ready_url: params['meta_bn-recording-ready-url'] }
```

Comment:
One detail here. I don't think it is documented but we need to make this backward compatible with the old meta parameters
See https://github.com/bigbluebutton/bigbluebutton/blob/develop/record-and-playback/core/scripts/post_publish/post_publish_recording_ready_callback.rb#L48-L51